core/src/main/java/org/p2p/core/token/TokenMetadata.kt

```diff
@@ -0,0 +1,6 @@
+package org.p2p.core.token
+
+data class TokenMetadata(
+    val timestamp: String,
+    val data: List<TokenData>
```

Comment:
can we rename class TokenData to TokenMetadata? 
this one can be TokenMetadataDetails or smth like that

i expected that we won't have TokenData anymore - TokenMetadata is better because it is what is it, not some random data